maps/tests/Metadata/setTiles.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <script>
+            var script = document.createElement('script');
+            // Don't do this at home kids! The "document.referrer" part is actually inserting a XSS security.
+            // We are OK in this precise case because the HTML page is hosted on the "maps" domain that contains only static files.
+            script.setAttribute('src', document.referrer + 'iframe_api.js');
```

Comment:
use script instead of cowebsite - iframe_api is directly injected

maybe with an onZoneEnter()